components/layout/Nav.js

```diff
@@ -80,11 +104,19 @@ const StyledNav = styled.nav`
       transform: translate(-50%, 0);
     }
   }
-  @media (max-width: 1300px) {
-    border-top: 1px solid var(--lightGray);
-    width: 100%;
-    justify-content: center;
-    font-size: 1.5rem;
+  #tutorialsLink {
+    @media (max-width: 390px) {
+      display: none;
+    }
+  }
+  #cart {
+    border: 2px solid var(--Primary);
+    border-radius: 50px;
+    margin-left: 2rem;
+    &:hover,
+    &:focus {
+      background-color: var(--PrimaryLight);
+    }
```

Comment:
Separate all of these classes, id's and elements into their own styled component